kitsune/motidings/tests/__init__.py

```diff
@@ -0,0 +1,10 @@
+from factory.django import DjangoModelFactory
+from tidings.models import Watch
+
+
+class WatchFactory(DjangoModelFactory):
+    class Meta:
+        model = Watch
+
+    event_type = 'fooevent'
+    is_active = True
```

Comment:
Is there any reason this factory shouldn't have make instances with a `secret` value by default?